module_documentation_sources/plugin_formatter.py

```diff
@@ -1,782 +0,0 @@
-# Copyright: (c) 2012, Jan-Piet Mens <jpmens () gmail.com>
```

Comment:
removing this `module_documentation_sources` is a good idea, but then we don't have any documentation for the ansible plugin anymore.
The best would be to upload the plugin to ansible-galaxy. If the documentation is well done (it should already be the case), it is automatically displays on the site of ansible galaxy.
what do you think ?
(also we should update the README)

Reply:
like you said, the documentation exists, it is embedded in the module code + it's easy to generate it with ansible-doc (text mode) + the user can generate it himself in html format with sphinx. Right now we do not provide the documentation, we provide a broken sphinx script to help the user create it... So I don't think it will be missed.
However I find your idea of uploading the module to ansible-galaxy very good.
I propose we work on that but it should not block this cleanup. Cleanup or not, the documentation still does not exists, is still up to the user to generate, and so the idea of using ansible-galaxy is good :)

Reply:
Seems good to me.
We should open an issue to upload the plugin in ansible galaxy, and remove the `module_documentation` part on the README.

 @dupremathieu are you okay with that ?